web/src/read_receipts.js

```diff
@@ -57,7 +57,7 @@ export function show_user_list(message_id) {
                                     {
                                         num_of_people: users.length,
                                         "z-link": (content_html) =>
-                                            `<a href="/help/read-receipts">${content_html.join(
+                                            `<a href="/help/read-receipts" target="_blank">${content_html.join(
```

Comment:
This needs `noreferrer noopener` added -- we do this for every other link with `_blank` on it, and it's important for security.

Reply:
> This needs `noreferrer noopener` added -- we do this for every other link with `_blank` on it, and it's important for security.

Fixed it.